.github/workflows/format.yml

```diff
@@ -0,0 +1,55 @@
+on: [push, pull_request]
+
+name: Format and Clippy
+
+jobs:
+  clippy:
+    name: Clippy
+    runs-on: ubuntu-latest
```

Comment:
We will collect CI time from PR to CI accept. Please let me know if you want some other data.

Good suggest to fix the ubuntu version. Will do that.

We have to use nightly version so far, because we are using cargo xbuild with - `cargo xbuild -p rust-tdshim --target x86_64-unknown-uefi --release`.

Reply:
Fix ubuntu version.